openlibrary/templates/type/edition/view.html

```diff
@@ -27,6 +26,9 @@
       $if not selected_ocaid or selected_ocaid == e.get('ocaid'):
         $ edition = e
 
+$if not work:
+  $ work = edition
+  $ work['edition_count'] = 1
```

Comment:
Would be great if we could avoid modifying the record. Line 40 looks like it sets `edition_count`; could we just set `edition_count` here in an if-else?

Reply:
It is not clear to me why an `edition` would have an `edition_count` and given that that `work.edition_count` is used on lines 40, 210, and 297 (the latter raises the exception), I found it necessary to create `work.edition_count`.  Would this alternative be more acceptable?

Reply:
I am also a bit confused by the nomenclature.  This is called an _orphaned work_ but it seems to be more of an _orphaned edition_ because the edition exists and the work does not. This code temporarily creates a virtual work from an edition.

Reply:
This case is actually on orphaned edition; I think it was just a slip up. Basically we should avoid editing `work` or `edition`; they should be considered read-only.  So if we can have a separate variable where we store the `edition_count` (or one in the case that it's an orphan), we should be good :+1:

Reply:
Is it possible for an edition to have more that one edition?